book/3-classification.tex

```diff
@@ -268,4 +268,74 @@ \subsection{Phi Coefficient}
 \clearpage
 \thispagestyle{classificationstyle}
 \section{MCC}
-\subsection{Matthew's Correlation Coefficient}
\ No newline at end of file
+\subsection{Matthew's Correlation Coefficient}
+
+
+% ---------- EC ----------
+\clearpage
+\thispagestyle{classificationstyle}
+\section{EC}
+\subsection{Expected Cost}
+
+The Expected Cost (EC) metric is designed to evaluate the performance of decision-making systems. It is the expected value of a cost function that assigns a penalty to each combination of the system's decision and the true class of a sample.
+
+\begin{center}
+\tikz{
+\node[inner sep=2pt, font=\Large] (a) {
+{
+$\displaystyle
+EC = \frac{1}{{\color{red}N}} \sum_{t=1}^{N} {\color{purple}C}({\color{cyan}h_t}, {\color{orange}d_t})
```

Comment:
use formula colors defined in the main.tex file: https://github.com/NannyML/The-Little-Book-of-ML-Metrics/blob/398fcaa29abb78bcf251143f4a13251df49c5e8a/book/main.tex#L39